Library/Homebrew/PATH.rb

```diff
@@ -1,3 +1,4 @@
+# typed: false
```

Comment:
What decides if a file is `false`/`ignore`/`true`?

Reply:
`ignore` does not show any errors, `false` shows syntax errors and `true` reports type errors. When `srb tc` is run for the very first time, it shows 0 errors (despite having some files as `true`).
IIUC, then `srb init` assigns each file, the highest degree of the `typed` sigil till it shows no errors initially. 
@issyl0 please confirm. Thank you! 😄

Reply:
Correct!